docs/installation/est/Installing_EST_pkispawn.md

```diff
@@ -0,0 +1,180 @@
+
+```
+est_ca_profile=estServiceCert
+est_ca_user_name=
+est_ca_user_password=
+est_ca_user_password_file=
+est_ca_user_certificate=
+est_realm_type=
+est_realm_custom=
+est_realm_url=
+est_realm_auth_type=BasicAuth
+est_realm_bind_dn=cn=Directory Manager
+est_realm_bind_password=
+est_realm_nickname=
+est_realm_user=
+est_realm_username=
+est_realm_password=
+est_realm_users_dn=ou=people,dc=est,dc=pki,dc=example,dc=com
+est_realm_groups_dn=ou=groups,dc=est,dc=pki,dc=example,dc=com
+est_realm_statements=/usr/share/pki/est/conf/realm/postgresql/statements.conf
+est_authorizer_exec_path=/usr/share/pki/est/bin/estauthz
+```
+
+The `est_ca_*` provides information related to the user and profile
+configured in the CA for the EST subsystem.
+
+The `est_authorizer_exec_path` is the executable responsible to verify
+the authorisation. The provided script checks only that the user has
+the role *estclient*.
+
+The `est_realm_*` options allow to customise the realm. Possible types
+are: ds, postgresql and in-memory. As an example, to install EST with
+PostgreSQL the command will be
+
+```
+pkispawn \
+    -f /usr/share/pki/server/examples/installation/est.cfg \
+    -s EST \
+    -D est_realm_url="jdbc:postgresql://postgresql.example.com:5432/est?ssl=true&sslmode=require" \
+    -D est_realm_type=postgresql \
+    -D est_realm_user=est \
+    -D est_realm_password=mysecretpassword \
+    -v
+```
+
+The `est_realm_custom` is a path to a custom realm configuration for
+tomcat and if provided it will overwrite all other realm related
+configurations.
+
+### Installation on separate instance
+
+When installed on a separate instance the CA URL and the signing
+certificate has to be provided. This is done with the command:
+
+```
+pkispawn \
+    -f /usr/share/pki/server/examples/installation/est.cfg \
+    -s EST \
+    -D est_realm_url=ldap://estds.example.com:3389 \
+    -D pki_ca_uri=https://ca.example.com:8443 \
+    -D pki_cert_chain_path=ca_signing.crt \
+    -D pki_cert_chain_nickname=caSigning \
+    -v
+```
+
+EST will get the SSL server certificate through the EST provided user
+and profile. If the SSL certificate (and the user certificate if it is
+needed to connect to the CA) should be created differently then it is
+possible to provide a PKCS12 bundle including the certificate and the
+full chain. It is important that the certificate aliases in PKCS12 match
+with the nickname used by EST. For SSL certificate the nickname
+configured in `est.cfg` is `sslserver` but can be modified. The
+command to deploy with the certificate bundle is:
+
+```
+pkispawn \
+    -f /usr/share/pki/server/examples/installation/est.cfg \
+    -s EST \
+    -D est_realm_url=ldap://estds.example.com:3389 \
+    -D pki_ca_uri=https://ca.example.com:8443 \
+    -D est_ca_user_password= \
+    -D est_ca_user_certificate=estUser \
+    -D pki_server_pkcs12_path=$SHARED/est_server.p12 \
+    -D pki_server_pkcs12_password=Secret.123 \
```

Comment:
We probably should explain how to create the `estUser` and `est_server.p12`, maybe in a separate section or a separate doc. The `$SHARED` should not be included since it's only available in the CI.

Reply:
I am adding a note but if it is not clear then it is better to move and add just a link